### PR TITLE
Provide support to enable read write ldap in toml

### PR DESCRIPTION
**Pull request by Nirothipan**
Fixes https://github.com/wso2/micro-integrator/issues/1800

### CODE REVIEW

File: distribution/src/resources/config-tool/deployment-full.toml [hunk 1]
```diff
@@ -47,7 +47,7 @@ properties.isCascadeDeleteEnabled = true    #inferred
 
 [user_store]
 #usr-mgt.xml
-type = "read_only_ldap" # inferred default read_only_ldap
+type = "read_only_ldap" # inferred default read_only_ldap # OR  read_write_ldap
 class = "org.wso2.micro.integrator.security.user.core.ldap.ReadOnlyLDAPUserStoreManager" # inferred
```
Comment (sdkottegoda) [inline distribution/src/resources/config-tool/deployment-full.toml:51]: let's remove this entry, may be in a separate commit. This is an advanced configuration.

Comment (Nirothipan) [reply distribution/src/resources/config-tool/deployment-full.toml:51]: since this is full toml , does it matters ?
